packages/flutter/lib/src/widgets/lazy_block.dart

```diff
@@ -124,7 +136,7 @@ class _RenderLazyBlock extends RenderVirtualViewport<_LazyBlockParentData> {
       child.parentData = new _LazyBlockParentData();
   }
 
-  double _noIntrinsicExtent() {
+  bool _debugThrowIfNotCheckingIntrinsics() {
     assert(() {
       if (!RenderObject.debugCheckingIntrinsics) {
         throw new UnsupportedError(
```

Comment:
This assertion text should probably be updated a little. :-)

Also, please remove the second \n while you're here.